### DISCUSSION TITLE
Translate Doc

### DISCUSSION THREAD

**Discussion by alirezakkt** [OWASP/mastg#2928] [Q&A] [answered]
Hello friends
I am a Junior developer
And I want to start translating documents and contributing into the open source world
Please guide me
I can translate mastg documents and make a pull request in to mastg ?

**Comment by sushi2k** [comment, marked as answer]
Thank you for reaching out and planning to volunteer @alirezakkt. We had translations in the past of the MASVS and MASTG, but decided a few years back against it. Please do see here the justification: https://mas.owasp.org/contributing/4_Add_new_Language/

Having said, we do have many open issues and you could work on one of these: https://github.com/OWASP/owasp-mastg/issues. Please let me know if you would like to work on one, then I can assign it to you.

**Comment by alirezakkt** [comment]
Thank you for your time. the answer you gave I am very motivated to
contribute to the owasp with the little knowledge I have and learn new
things at the same time. Thank you for giving me this opportunity. I will
do my best.
